tests/unit/preprocessor/_derive/test_xco2.py

```diff
@@ -0,0 +1,58 @@
+"""Test derivation of ``xco2``."""
+import iris
+import numpy as np
+import pytest
+
+import esmvalcore.preprocessor._derive.xco2 as xco2
+
+from .test_shared import get_cube
+
+
+@pytest.fixture
+def cubes():
+    """Input cubes for derivation of ``xco2``."""
+    co2_cube = get_cube([[[[1.0]], [[2.0]]]], air_pressure_coord=True,
+                        standard_name='mole_fraction_of_carbon_dioxide_in_air',
+                        var_name='co2', units='1e-6')
+    hus_cube = get_cube([[[[0.2]], [[0.2]]]], air_pressure_coord=True,
+                        standard_name='specific_humidity', var_name='hus',
+                        units='%')
+    zg_cube = get_cube([[[100.0]]], air_pressure_coord=False,
+                       standard_name='geopotential_height', var_name='zg',
+
+                       units='m')
+    ps_cube = get_cube([[[100000.0]]], air_pressure_coord=False,
+                       standard_name='surface_air_pressure', var_name='ps',
+                       units='Pa')
+    return iris.cube.CubeList([co2_cube, hus_cube, zg_cube, ps_cube])
+
+
+def test_xch4_calculate(cubes):
+    """Test function ``calculate``."""
```

Comment:
Should this here be called "test_xco2_calculate(cubes):"?
Or are you just adding the unit test for xco2 to the xch4 unit test?